factcast-schema-registry-cli/src/main/resources/site/hugo-theme-docdock-master/exampleSite/content/shortcode/icon.md

```diff
@@ -0,0 +1,1259 @@
++++
+draft = false
+title = "icon"
+description = "Display an icon."
+
+[menu.main]
+parent = "shortcodes"
+identifier = "icon"
++++
+
+Display an icon like `{{% icon cloud %}}`
+
+It uses :
+
+* glyphicons library (bootstrap).[more info here](http://getbootstrap.com/components/#glyphicons)
+* fontawesome library [more info here](http://fontawesome.io/)
+
+This `icon` shortcode will display an icon in your page. 
+
+## Usage
+
+| Parameter | Default | Description |
+|:--|:--|:--|
+| name | **required** | name of icon (see bellow) |
+| size | none | size of icon, medium, xx-small, x-small, small, large, x-large, xx-large, 11px, 2em, 20%.... |
+
+{{%alert info%}}
+**Tips :**
+setting only the name as argument works too : `{{</*icon film*/>}}` instead of `{{</*icon name="film"*/>}}`
+{{%/alert%}}
+
+# Demo
+	
+	{{</* icon name="film" size="large" */>}}
+
+{{<icon name="film" size="large">}}
+
+.
+
+# icons available
+
+- {{< icon name="asterisk" size="large" >}} asterisk
+- {{< icon name="plus" size="large" >}} plus
+- {{< icon name="euro" size="large" >}} euro
+- {{< icon name="eur" size="large" >}} eur
+- {{< icon name="minus" size="large" >}} minus
+- {{< icon name="cloud" size="large" >}} cloud
+- {{< icon name="envelope" size="large" >}} envelope
+- {{< icon name="pencil" size="large" >}} pencil
+- {{< icon name="glass" size="large" >}} glass
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [[unordered-list-marker-style] Marker style should be `*`](https://app.codacy.com/manual/uwe/factcast/pullRequest?prid=4955738)